k3s_camera/README.md

```diff
@@ -0,0 +1,10 @@
+With the k3s device deployment, we do not have additional mounts in containers.
+This means the containers cannot use the device's `/opt/vc` directory.
+
+As a solution, we can manually copy the precompiled `/opt/vc` into the image.
+Precompiled binaries can be downloaded here:
+https://github.com/raspberrypi/firmware
```

Comment:
Are they precompiled under 64bit or 32bit?

Reply:
They are 32 bit, which is why I suspect this does not work on the 64 bit image.